server/model/run.go

```diff
@@ -0,0 +1,74 @@
+package model
+
+import (
+	"math"
+	"time"
+
+	"github.com/kubeshop/tracetest/server/traces"
+)
+
+var (
+	Now = func() time.Time {
+		return time.Now()
+	}
+)
+
+func (r Run) Copy(testVersion int) Run {
+	r.Results = nil
+	r.Trace = nil
```

Comment:
Trace should be kept. This is being called by the `rerun` command and today it doesn't (and shouldn't) retrigger the transaction to run the test, it assumes the trace didn't change and only re-execute the assertion engine using the latest test definition.